src/test/java/io/deephaven/csv/CsvReaderTest.java

```diff
@@ -1873,6 +1870,320 @@ public void colnumPassedThrough() throws CsvReaderException {
         Assertions.assertThat(bh2Num).isEqualTo(2);
     }
 
+    /**
+     * Addresses <a href="https://github.com/deephaven/deephaven-csv/issues/212"> A user requested that the library be
+     * able to read files like this.
+     */
+    @Test
+    public void bug212() throws CsvReaderException {
+        final String input =
+                ""
+                        + "NAME                     STATUS       AGE      LABELS\n"
+                        + "argo-events              Not Active   2y77d    app.kubernetes.io/instance=argo-events,kubernetes.io/metadata.name=argo-events\n"
+                        + "argo-workflows           Active       2y77d    app.kubernetes.io/instance=argo-workflows,kubernetes.io/metadata.name=argo-workflows\n"
+                        + "argocd                   Active       5y18d    kubernetes.io/metadata.name=argocd\n"
+                        + "beta                     Not Active   4y235d   kubernetes.io/metadata.name=beta\n";
+
+        final CsvSpecs specs = defaultCsvBuilder().hasFixedWidthColumns(true)
+                .ignoreSurroundingSpaces(true).build();
```

Comment:
ignoreSurroundingSpaces defaults to true; is false a legit value? If not, we should err if it is set.

Reply:
I've removed a bunch of places where I was redundantly setting it to true. Setting it to false *does* work but is not IMO very useful. Unit test in `CsvReaderTest#fixedColumnsMayIncludeOrExcludeSurroundingSpaces`